test/python/quantum_info/states/test_stabilizerstate.py

```diff
@@ -455,10 +539,51 @@ def test_probabilities_dict_qubits(self):
                     "111": 0.125,
                 }
                 self.assertEqual(value, target)
+                StabilizerStateTestingTools._verify_individual_bitstrings(
+                    self, target, stab, decimals=decimals
+                )
                 probs = stab.probabilities(decimals=3)
                 target = np.array([0.125, 0.125, 0.125, 0.125, 0.125, 0.125, 0.125, 0.125])
                 self.assertTrue(np.allclose(probs, target))
 
+    @combine(num_qubits=[5, 6, 7, 8, 9])
+    def test_probabilities_dict_hgate_medium_num_qubits(self, num_qubits):
+        """Test probabilities_dict_from_bitstring methods with medium number of qubits that are still
+        reasonable to calculate the full dict with probabilities_dict of all possible outcomes"""
+        qc: QuantumCircuit = QuantumCircuit(num_qubits)
+        for qubit_num in range(0, num_qubits):
+            qc.h(qubit_num)
+        stab = StabilizerState(qc)
+
+        expected_result: float = float(1 / (2**num_qubits))
+        target_dict: dict = StabilizerStateTestingTools._bitstring_product_dict(num_qubits)
+        target_dict.update((k, expected_result) for k in target_dict)
+        for _ in range(self.samples):
+            with self.subTest(msg="P(None)"):
+                value = stab.probabilities_dict()
+                self.assertDictEqual(value, target_dict)
+                StabilizerStateTestingTools._verify_individual_bitstrings(self, target_dict, stab)
+                probs = stab.probabilities()
+                target = np.array(([expected_result] * (2**num_qubits)))
+                self.assertTrue(np.allclose(probs, target))
+
+    @combine(num_qubits=[25, 50, 100, 200])
```

Comment:
it seems that `test_stabilizerstate` takes about twice more time now. I'm not sure with tests are responsible to this, but probably we can do without the very large numbers of qubits.

Reply:
it would be nice if you could think of some test where the probabilities are not all equal (and it's not a GHZ state)